keps/693-multikueue/README.md

```diff
@@ -205,7 +217,15 @@ type KubeConfig struct {
     LocationType LocationType `json:"locationType"`
 }
 
-type MultiKueueClusterStatus {
+type ClusterProfile struct {
```

Comment:
Please also add here the Config API, as I see it introduced in the implementation PR.